taxonomies/ingredients.txt

```diff
@@ -15442,6 +15446,10 @@ fr:chipolatas
 <en:pork meat
 fr:chipolatas aux herbes
 
+<en:pork skin
```

Comment:
This would mean the "en:pork-skin" entry is a parent of itself.

Reply:
ok, changed to pork